app/src/main/java/in/testpress/testpress/core/ProductService.java

```diff
@@ -19,6 +23,15 @@
     @GET("/{product_url}")
     ProductDetails getProductDetails(@EncodedPath("product_url") String productUrlFrag, @Header("Authorization") String authorization);
 
+    @POST(Constants.Http.URL_ORDERS_FRAG)
+    Order order(@Body HashMap<String, Object> arguments, @Header("Authorization") String authorization);
+
+    @PUT(Constants.Http.URL_ORDERS_FRAG+"{order_id}"+"/confirm/")
```

Comment:
This is not the place to calculate the url. It is better to pass it like product_url.